### PR TITLE
never download target version if feature flag gets revoked; optimize …

### PR DESCRIPTION
**Pull request by nickpell**
…performance for domains without flag

### CODE REVIEW

File: corehq/apps/app_manager/views/releases.py [hunk 1]
```diff
@@ -103,7 +103,11 @@ def paginate_releases(request, domain, app_id):
     for app in saved_apps:
         app['include_media'] = app['doc_type'] != 'RemoteApp'
         app['j2me_enabled'] = app['menu_item_label'] in j2me_enabled_configs
-        app['target_commcare_flavor'] = SavedAppBuild.get(app['_id']).target_commcare_flavor
+        app['target_commcare_flavor'] = (
+            SavedAppBuild.get(app['_id']).target_commcare_flavor
+            if toggles.TARGET_COMMCARE_FLAVOR.enabled(domain)
+            else 'none'
```
Comment (pr33thi) [inline corehq/apps/app_manager/views/releases.py]: How come it's a string `'none'` instead of `None`?

Comment (nickpell) [reply corehq/apps/app_manager/views/releases.py]: ```target_commcare_flavor``` has a default value: https://github.com/dimagi/commcare-hq/blob/ccece710d474616f6bd0901c56790c58576f7f72/corehq/apps/app_manager/models.py#L4727

**Review by proteusvacuum — APPROVED** [review]

File: corehq/apps/app_manager/views/releases.py (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/app_manager/views/releases.py]: Agree it's weird, but it's consistent with other app profile settings that use `none` as a value.

Comment (proteusvacuum) [reply corehq/apps/app_manager/views/releases.py]: YOLO

**Pull request merged**